samples/app-cache-meetings/nodejs/src/components/configure.jsx

```diff
@@ -0,0 +1,37 @@
+// <copyright file="configure.jsx" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// </copyright>
+
+import { useEffect } from "react";
+import { Text } from "@fluentui/react-northstar";
+import * as microsoftTeams from "@microsoft/teams-js";
+
+// Configure page.
+const Configure = props => {
+    useEffect(() => {
+        microsoftTeams.app.initialize().then(() => {
+            microsoftTeams.app.notifySuccess();
+
+            microsoftTeams.pages.config.registerOnSaveHandler(function (saveEvent) {
+                microsoftTeams.pages.config.setConfig({
+                    entityID: "AppCache",
```

Comment:
this is not a useful entityId. generate a random id so that each tab instance has a unique ID and  ensure the cached launch prints out the entityID

Reply:
generating entity id for each tab instance and printing the entity Id